saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/ace/ACEReader.scala

```diff
@@ -1,123 +0,0 @@
-        val span = s.getStartSpan -> s.getEndSpan
-        Sentence(s.getSentenceId, ms, toks, span)
-      })
-      val doc = Document(id, sents)
-      docs += doc
-    }
-    docs
-  }
-
-  def readAll(baseDir: String = "data/ace/ace-2005/data/cache/"): Seq[Document] =
-    new File(baseDir).list().map(baseDir + _).flatMap(f => read(f))
-}
-
-object TestACEReader extends App {
-  (new IllinoisReader).readAll("/Users/sameer/Work/data/ace/ace-2005/data/cache/")
-}
-
-object AnnotateACE extends App {
-
-  def annotateAllDocument(functor: AceFileProcessor, inputFolderStr: String, outputFolderStr: String, dtdFile: File) {
-    val var20 = new File(inputFolderStr)
-    val subFolderList = var20.listFiles().filter(f => f.getName == "nw")
-
-    for (folderIndex <- 0 until subFolderList.length) {
-      val filter = new FilenameFilter() {
-        def accept(directory: File, fileName: String) = fileName.endsWith(".apf.xml")
-      };
-      val subFolderEntry = subFolderList(folderIndex)
-      val labelFolder = new File(subFolderEntry.getAbsolutePath() + "/adj");
-      val fileList = labelFolder.listFiles(filter);
-      Files.copy(Paths.get(dtdFile.getAbsolutePath), Paths.get(labelFolder.getAbsolutePath + "/" + dtdFile.getName), StandardCopyOption.REPLACE_EXISTING)
-
-      for (fileID <- 0 until fileList.length) {
-        val annotationFile = fileList(fileID).getAbsolutePath();
-        System.err.println("reading ace annotation from \'" + annotationFile + "\'...");
-        var annotationACE: ACEDocumentAnnotation = null;
-
-        try {
-          annotationACE = ReadACEAnnotation.readDocument(annotationFile);
-        } catch {
-          case var19: XMLException => var19.printStackTrace(); System.exit(1)
-        }
-
-        val outputFile = new File(outputFolderStr + annotationACE.id + ".ta");
-        if (!outputFile.exists()) {
-          if (annotationFile.contains("rec.games.chess.politics_20041216.1047")) {
-            System.out.println("[DEBUG]");
-          }
-
-          System.out.println("[File]" + annotationFile);
-          val aceDoc = functor.processAceEntry(subFolderEntry, annotationACE, annotationFile);
-
-          try {
-            val f = new FileOutputStream(outputFile);
-            val e = new ObjectOutputStream(f);
-            e.writeObject(aceDoc);
-            e.flush();
-          } catch {
-            case var18: Exception => var18.printStackTrace(); System.exit(1)
-          }
-        }
-      }
-    }
-
-  }
-
-  val baseDir = "/Users/sameer/Work/data/ace/ace-2005/data/"
-  val dtdFile = "/Users/sameer/Work/data/ace/ace-2005/dtd/apf.v5.1.1.dtd"
-  val docDirInput = baseDir + "English/"
-  val docDirOuput = baseDir + "cache/"
-  new File(docDirOuput).mkdirs
-  val taBuilder = new CcgTextAnnotationBuilder(new IllinoisTokenizer())
-  val functor = new AceFileProcessor(taBuilder)
-  annotateAllDocument(functor, docDirInput, docDirOuput, new File(dtdFile))
-}
```

Comment:
They are never used in any applications (not even in [RE application](https://github.com/IllinoisCogComp/saul/pull/350/files))